dask_sql/physical/rex/core/call.py

```diff
@@ -196,14 +196,16 @@ def cast(self, operand, rex=None) -> SeriesOrScalar:
         if not is_frame(operand):
             return operand
 
-        output_type = str(rex.getType())
-        output_type = sql_to_python_type(output_type.upper())
-
-        return_column = cast_column_to_type(operand, output_type)
+        sql_output_type = str(rex.getType())
+        python_output_type = sql_to_python_type(sql_output_type.upper())
+        return_column = cast_column_to_type(operand, python_output_type)
 
         if return_column is None:
             return operand
         else:
+            # handle datetime type specially
```

Comment:
Same as above - just adding a link to https://github.com/dask-contrib/dask-sql/issues/296

```suggestion
            # address datetime format mismatch
            # https://github.com/dask-contrib/dask-sql/issues/296
```